libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -187,9 +204,17 @@ bool AP_Arming::airspeed_checks(bool report)
             return true;
         }
         for (uint8_t i=0; i<AIRSPEED_MAX_SENSORS; i++) {
-            if (airspeed->enabled(i) && airspeed->use(i) && !airspeed->healthy(i)) {
-                check_failed(ARMING_CHECK_AIRSPEED, report, "Airspeed %d not healthy", i + 1);
-                return false;
+            if (airspeed->enabled(i)) {
+                if (!airspeed->healthy(i)) {
+                    check_failed(ARMING_CHECK_AIRSPEED, report, "Airspeed %d not healthy", i + 1);
+                    return false;
+                } else if (airspeed->get_airspeed() > 5.01f) {
+                    check_failed(ARMING_CHECK_AIRSPEED, report, "Airspeed %d unexpected high airspeed", i + 1);
+                    return false;
+                } else if (airspeed->get_airspeed() < 0.05f) {
```

Comment:
and this means you can't takeoff in still conditions. Not all sensors produce that much noise.

Reply:
So can be set as warnings not failures?
And keep the checking if enabled change?

Reply:
It's a false positive warning in far to many cases though, and will lead to more support headaches then it will save planes as everyone will be trained to ignore it.

Reply:
But I could replicate the case where ARSPD_BIN is wrong and it's not working as expected. I can change it to smaller value.